pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -81,86 +84,80 @@ var (
 
 // Reconciler implements controller.Reconciler for Configuration resources.
 type Reconciler struct {
-	*reconciler.Base
+
+	// KubeClientSet allows us to talk to the k8s for core APIs
+	KubeClientSet kubernetes.Interface
+
+	// TriggersClientSet allows us to configure triggers objects
+	TriggersClientSet triggersclientset.Interface
+
+	// CachingClientSet allows us to instantiate Image objects
+	CachingClientSet cachingclientset.Interface
```

Comment:
I don't see any usage of `CachingClientSet` can we remove this?

Reply:
Good catch :) I think this is a left over from copying over from the Pipelines controller

Reply:
Turns out removing this also removes some dependencies on knative/pkg/caching!